api/routers/legacy_debug.py

```diff
@@ -181,7 +197,7 @@ async def sql_get_contributions(contributors: List):
         JOIN Players as pl on (pl.id = rs.reportingID)
         join Players as ban on (ban.id = rs.reportedID)
         WHERE 1=1
-            AND pl.name in :contributors
+            AND pl.normalized_name in :contributors
```

Comment:
Make sure this route also contains the necessary recommendations as listed in the previous route.

Reply:
I added in jagexify_names_list. I think that name is really dumb, but it works! I'm not sure what else to call it.

Reply:
Sounds good!